src/Workspaces/CSharp/Portable/Extensions/SyntaxTreeExtensions.cs

```diff
@@ -53,6 +53,7 @@ internal static partial class SyntaxTreeExtensions
                     case SyntaxKind.VolatileKeyword:
                     case SyntaxKind.UnsafeKeyword:
                     case SyntaxKind.AsyncKeyword:
+                    case SyntaxKind.RefKeyword:
```

Comment:
💭 Why are `InKeyword`, `OutKeyword`, and `ThisKeyword` not part of this?

Reply:
those are not considered modifiers in the sense of how this function is working.  This is for the modifiers you get on a member or type decl.  The method could have its name update accordingly though.